ci/scripts/jenkins/checks.sh

```diff
@@ -20,15 +20,33 @@ source ${WORKSPACE}/ci/scripts/jenkins/common.sh
 
 fetch_base_branch
 
-conda activate rapids
+gpuci_logger "Creating conda env"
+rm -rf ${MORPHEUS_ROOT}/.cache/ ${MORPHEUS_ROOT}/build/
+conda config --add pkgs_dirs /opt/conda/pkgs
+conda config --env --add channels conda-forge
+conda config --env --set channel_alias ${CONDA_CHANNEL_ALIAS:-"https://conda.anaconda.org"}
+mamba env create -q -n morpheus -f ${MORPHEUS_ROOT}/docker/conda/environments/cuda${CUDA_VER}_dev.yml
+conda activate morpheus
 
 gpuci_logger "Installing CI dependencies"
-mamba install -q -y -c conda-forge "yapf=0.32"
+mamba env update -q -f ${MORPHEUS_ROOT}/docker/conda/environments/cuda${CUDA_VER}_ci.yml
 
 show_conda_info
 
 gpuci_logger "Runing Python style checks"
 ${MORPHEUS_ROOT}/ci/scripts/python_checks.sh
 
+gpuci_logger "Configuring cmake for Morpheus"
+cmake -B build -G Ninja ${CMAKE_BUILD_ALL_FEATURES} -DCCACHE_PROGRAM_PATH=$(which sccache) .
+
+gpuci_logger "Building targets that generate source code"
+cmake --build build --target style_checks --parallel ${PARALLEL_LEVEL}
+
+gpuci_logger "sccache usage for source build:"
+sccache --show-stats
+
+gpuci_logger "Runing C++ style checks"
+SKIP_IWYU=1 ${MORPHEUS_ROOT}/ci/scripts/cpp_checks.sh
```

Comment:
Why are you skipping IWYU here?

Reply:
Because IWYU isn't installed int the container, and the headers haven't been updated to pass an IWYU check. 

I spun the IWYU bit off into another PR #287

Reply:
Ok, makes sense with the other PR